swiftgen-cli/ArgumentsUtils.swift

```diff
@@ -62,6 +70,15 @@ enum OutputDestination: ArgumentConvertible {
   }
 }
 
+// MARK: Change comparsion
+
+extension Path {
+  func isContentEqualTo(string: String) throws -> Bool {
```

Comment:
Just nitpicking here, but I have to admit I feel like the name of this method sounds a bit Objective-C-y to me?
Open to suggestions as to what would be a better Swift name for it though.